src/ExternalConnector.cpp

```diff
@@ -244,7 +244,7 @@ char *command_completion(const char *text, int state)
 	}
 
 	wxString test(wxString(text, *wxConvCurrent).Lower());
-	while (nextCommand != curCommands->end()) {
+	while (curCommands && (nextCommand != curCommands->end())) {
```

Comment:
The documentation for `rl_completion_matches` says that _entry_func_ (i.e. `command_completion()` in this case) will first be called with _state_ being zero on the first call, and non-zero on subsequent calls. Thus `curCommands` (and `nextCommand`, too) will always be initialized and non-zero on calls where _state_ is non-zero.